servers/src/main/java/tachyon/web/WebInterfaceWorkersServlet.java

```diff
@@ -126,19 +132,20 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
    * @param workerInfos The list of WorkerInfo objects
    * @return The list of NodeInfo objects
    */
-  private NodeInfo[] generateOrderedNodeInfos(List<WorkerInfo> workerInfos) {
+  private NodeInfo[] generateOrderedNodeInfos(Collection<WorkerInfo> workerInfos) {
```

Comment:
Using `Collection` instead of `List` ad input param, allows the method to be more generic, while keeping the same execution complexity.